Cargo.toml

```diff
@@ -16,16 +17,19 @@ name = "tikv"
 [[bin]]
 name = "tikv-server"
 
-[[bin]]
-name = "bench-tikv"
-path = "benches/bin/bench-tikv.rs"
-
 [[bin]]
 name = "tikv-ctl"
 
 [[bin]]
 name = "tikv-importer"
 
+[[bench]]
+name = "raftstore"
+harness = false
```

Comment:
why only harness for raftstore?

Reply:
`harness = false` means we self manages the benchmark process for this benchmark. For `benches.rs`, we use `#[bench]` and relies on the harness to trigger these benchmarks, so it does not have `harness = false`. However for `raftstore.rs`, since we use criterion.rs to run our dynamic benchmarks, we should make it `harness = false`.